.gitlab-ci.yml

```diff
@@ -205,15 +238,37 @@ build:build-project:
 
     # zip it
     - Compress-Archive -Path "$BUILD_MARKET_PLACE_PATH_UE5" -DestinationPath "$BUILD_MARKET_PLACE_PATH_UE5.zip"
+    
```

Comment:
I wonder if we should add a ticket to make some of this duplicated script code into a local method that can be called twice with the correct UE5/UE5.1 build parameters? I think that would be a good tool sharpening ticket to have for our next sprint.